.gitignore

```diff
@@ -14,5 +14,7 @@ npm-debug.log*
 /backstop_data/ci_report/
 /backstop_data/backstop_scenarios.json
 /backstop_data/packageDependentMap.json
+backstop.json
```

Comment:
I think we should still remove those from our gitignore files like we chatted in an earlier discussion.

Reply:
New commit resolves this. https://github.com/adobe/spectrum-css/commit/70145a937c2c0239fcea28987a1054856aef0249